internal/server/statvar/fetcher/svg_fetcher.go

```diff
@@ -165,7 +165,11 @@ func FetchAllSVG(
 			}
 			result[svg].ChildStatVars = append(
 				result[svg].ChildStatVars,
-				&pb.StatVarGroupNode_ChildSV{Id: sv, DisplayName: name},
+				&pb.StatVarGroupNode_ChildSV{
+					Id:          sv,
+					DisplayName: name,
+					SearchNames: []string{name},
```

Comment:
Do you have access to the SV descriptions? If so, consider adding it to `SearchNames`.

Also, it'll be good to start adding tests for custom dc functionality.

Reply:
added description and golden tests